### DISCUSSION TITLE
Fullscreen by Default

### DISCUSSION THREAD

**Discussion by bijanbina** [rustdesk/rustdesk#5522] [General]
### Bug Description

Is there a way to always open certain or all connections in full-screen mode?

### Expected Behavior

option to enable Fullscreen by Default

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.2.1

**Comment by dignow** [comment]
https://github.com/rustdesk/rustdesk/pull/5625

@bijanbina Hi, I do not add an option about "Fullscreen by Default".
But if you leave the session in the fullscreen mode,  The connecting to the same peer will enter the fullscreen mode.

**Comment by bijanbina** [comment]
Just saw that. That's even better. Thanks!
Can you checkout this bug as well?
https://github.com/rustdesk/rustdesk/discussions/5621
It's the biggest bug I have with the software!

**Comment by southerndoc** [comment]
When using a Windows computer connecting to another client, it seems the Windows computer remembers that I prefer connections to be full screen.  Each time I open the window it's always full screen.

With a Mac computer, it always opens to a regular screen.  Is there a way for it to remember or at least have an option to always open full screen?

Not sure if this is a bug with 1.4.0 or if it's a limitation not yet developed.
